nevergrad/benchmark/experiments.py

```diff
@@ -1073,6 +1074,53 @@ def rocket(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                             yield xp
 
 
+@registry.register
+def gym_multi(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Gym simulator. Maximize reward.
+    Budget 25, 50, ..., 1600.
+    Sequential or 30 workers."""
+    env_names = GymMulti().env_names()
+    seedg = create_seed_generator(seed)
+    optims = get_optimizers("basics", "progressive", "splitters", "baselines", seed=next(seedg))
+    for budget in [50, 100, 200, 400, 800, 1600]:
+        for func in [
+            GymMulti(name, control, neural_factor)
+            for control in [
+                "conformant",
+                "linear",
+                "neural",
+                "noisy_neural",
+                "noisy_scrambled_neural",
+                "scrambled_neural",
+            ]
```

Comment:
please build functions before the budget, this will be faster

Reply:
Done.